assets/js/buttons-examples.js

```diff
@@ -169,6 +169,10 @@ $(document).ready(function () {
           classes: "weibo",
           text: "Login with Weibo",
         },
+        {
+          classes: "github",
```

Comment:
Place this button in the correct alphabetical order.

Reply:
Done! have committed the changes.